src/main/java/seedu/address/logic/commands/TotalOrdersCommand.java

```diff
@@ -14,7 +14,7 @@
             + "Example: " + COMMAND_WORD;
 
     public static final String SHOWING_TOTAL_ORDERS_MESSAGE = "Opened/Refreshed total orders window.\n"
-            + "You might need to click the icon on taskbar if you have minimized it.";
+            + "You might need to restore the window if you have minimized it.";
```

Comment:
You might want to add this message to the help command as well if you add it here.

Reply:
Okay